TYPO3.Flow/Classes/TYPO3/Flow/Aop/Builder/ProxyClassBuilder.php

```diff
@@ -360,13 +362,21 @@ protected function buildAspectContainer($aspectClassName)
         }
         $introduceAnnotation = $this->reflectionService->getClassAnnotation($aspectClassName, \TYPO3\Flow\Annotations\Introduce::class);
         if ($introduceAnnotation !== null) {
-            if ($introduceAnnotation->interfaceName === null) {
-                throw new \TYPO3\Flow\Aop\Exception('The interface introduction in class "' . $aspectClassName . '" does not contain the required interface name).', 1172694761);
+            if ($introduceAnnotation->interfaceName === null && $introduceAnnotation->traitName === null) {
+                throw new \TYPO3\Flow\Aop\Exception('The introduction in class "' . $aspectClassName . '" does neither contain an interface name nor a trait name, at least one is required.', 1172694761);
```

Comment:
ah, sorry, I missed that I just extended the old extension instead of adding another one. I think I would leave with with the old code in that case. I thought I duplicated a code there.